<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ShowPreviousDays">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowPrevDays</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowTillBirthday">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>450</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowTillBday</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowIsLeap">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowIsLeap</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>951</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <row>
     <property name="text">
      <string>Date1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Date2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Date3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Date4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Date5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Date6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Date7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Date8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Date9</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Next Day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Previous Day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Is Leap</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Week Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Till Birthday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Duration</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="ShowNextDays">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowNextDays</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChooseFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowWeekNumbers">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>450</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowWeekNmbr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Text">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter your birth date(optional):</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Enter">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>490</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Wanna see table?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowDurations">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>450</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowDurations</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterBirthday">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>961</width>
      <height>621</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeFile">
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>30</y>
      <width>471</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add New Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>431</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Delete one of dates</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>161</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Enter Date to delete:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Delete this Date</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>Table</zorder>
   <zorder>ShowNextDays</zorder>
   <zorder>ChooseFile</zorder>
   <zorder>ShowWeekNumbers</zorder>
   <zorder>Text</zorder>
   <zorder>Enter</zorder>
   <zorder>ShowDurations</zorder>
   <zorder>EnterBirthday</zorder>
   <zorder>ShowPreviousDays</zorder>
   <zorder>ShowTillBirthday</zorder>
   <zorder>ShowIsLeap</zorder>
   <zorder>ChangeFile</zorder>
   <zorder>pushButton</zorder>
   <zorder>formLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
